Services/CO.CDP.Organisation.WebApi.Tests/UseCase/UpdateOrganisationUseCaseTest.cs

```diff
@@ -150,6 +151,77 @@ await act.Should()
             .ThrowAsync<InvalidUpdateOrganisationCommand.MissingAdditionalIdentifiers>();
     }
 
+    [Fact]
+    public async Task ShouldThrowInvalidUpdateOrganisationCommand_WhenIdentifierAlreadyExists()
+    {
+        // Arrange
+        var command = new UpdateOrganisation
+        {
+            Type = OrganisationUpdateType.AdditionalIdentifiers,
+            Organisation = new OrganisationInfo
+            {
+                AdditionalIdentifiers = [new OrganisationIdentifier
+                {
+                    Id = "13294342",
+                    LegalName = "Tcme",
+                    Scheme = "VAT"
+                }]
+            }
+        };
+
+        var organisation = Organisation;
+
+        _organisationRepositoryMock.Setup(repo => repo.Find(_anotherOrganisationId)).ReturnsAsync(AnotherOrganisation);
+
+        _organisationRepositoryMock.Setup(repo => repo.FindByIdentifier("VAT", "13294342")).ReturnsAsync(organisation);
+
+
+
+        Func<Task> act = async () => await UseCase.Execute((_anotherOrganisationId, command));
+
+
+        await act.Should()
+          .ThrowAsync<InvalidUpdateOrganisationCommand>()
```

Comment:
This should be a more specific exception. 

Then, there's no need to check the exception message if we already verify the type.